src/github.com/getlantern/flashlight/pubsub/pubsub.go

```diff
@@ -0,0 +1,24 @@
+// package pubsub provides a global publish and subscribe infrastructure
+// for lantern
+package pubsub
+
+import (
+	"reflect"
+
+	"github.com/asaskevich/EventBus"
+)
+
+var (
+	bus = EventBus.New()
+)
+
+// Pub publishes the given interface to any listeners for that interface.
+func Pub(topic interface{}) {
```

Comment:
Telling topics by type, while it saves an argument in Pub calls, seems more odd and limiting to me than the convenience warrants.  Exposing a geolookup.Topic string constant and using that for Pub and Sub calls seems cleaner to me.